src/vmm/src/vstate/vcpu/aarch64.rs

```diff
@@ -15,52 +14,37 @@ use versionize_derive::Versionize;
 use vm_memory::{Address, GuestAddress, GuestMemoryMmap};
 
 use crate::arch::aarch64::regs::Aarch64Register;
+use crate::arch::regs::{
+    get_mpstate, read_mpidr, restore_registers, save_core_registers, save_registers,
+    save_system_registers, set_mpstate,
+};
 use crate::vcpu::VcpuConfig;
 use crate::vstate::vcpu::VcpuEmulation;
 use crate::vstate::vm::Vm;
 
 /// Errors associated with the wrappers over KVM ioctls.
-#[derive(Debug)]
+#[derive(Debug, thiserror::Error)]
```

Comment:
Can this be extracted into a separate commit?

Reply:
made in a separate commit